client/js/views/GameView.vue

```diff
@@ -1253,30 +1249,68 @@ export default {
 .history-container {
   grid-area: history;
 }
+
 #field-right {
   height: 100%;
+  padding-top: 0.5rem;
 
   #history,
   #card-preview {
     margin: 0 auto;
-    padding: 5px;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    border: 1px solid #fff;
+    padding: 0.5rem;
     width: 80%;
     max-width: 80%;
     height: 100%;
+    border-radius: 20px;
+    box-shadow: 0px 0px 10px #111111;
   }
   #history {
-    background-color: rgba(0, 0, 0, 0.46);
+    background-color: rgba(241, 200, 160, 0.65);
+    color: #111111;
```

Comment:
@itsalaidbacklife A note for a future ticket, we may want to start looking into storing some of these hex values as color variables in scss files to get a true design system going.

Reply:
@peterancourt spot on, I would love to get a CSS config file of all of our base cuttle values in a `:root` rule block. I'm torn on Vuetify, I love it but would love to move away from it for some cuttle specific stuff since we're talking about pulling back on our global usages.